fuzz/fuzz_targets/minreq_http.rs

```diff
@@ -0,0 +1,59 @@
+
+extern crate jsonrpc;
+
+#[cfg(not(jsonrpc_fuzz))]
+compile_error!("You must set RUSTFLAGS=--cfg=jsonrpc_fuzz to run these test, or run the actual fuzz harness.");
```

Comment:
In 9710de91fd51f43aa1ac25dfd3c42e240ecb78b6:

These cause `cargo test` to fail, which will cause `cargo test --all` to fail once we move the fuzztests into a workspace.

Also, could we move the fuzztests into a workspace?

IMO we should just invert the `jsonrpc_fuzz` cfg variable and require the user *set* a cfg flag to get normal fuzzing. It's probably a bad idea to be fuzzing something with network access anyway.

Reply:
I just rebased this PR on top of #98 to make sure I got everything right. #98 is kinda trivial so you could either just merge this one and I"ll comment as such on #98 or else merge #98 if you prefer and I'll rebase this one - either way is good with me.